packages/ref-data-uploaders/txc-uploader/txc_processor.py

```diff
@@ -246,8 +249,47 @@ def iterate_through_journey_patterns_and_run_insert_queries(
     if admin_area_codes:
         insert_admin_area_codes(cursor, admin_area_codes, operator_service_id)
 
-def insert_admin_area_codes(cursor: aurora_data_api.AuroraDataAPICursor, area_codes, service_id):
-    codes_dict = {f"k{k}":v for k,v in enumerate(area_codes)}
+
+def collect_track_data(route_sections):
+    routes = []
+    for route_section in route_sections:
+        route_links = make_list(route_section["RouteLink"])
+        for route_link in route_links:
+            trackData = route_link.get("Track", None)
+            if trackData is not None:
+                tracks = trackData if isinstance(trackData, list) else [trackData]
+                for track in tracks:
+                    locations = make_list(track["Mapping"]["Location"])
+                    for location in locations:
+                        longitude = location["Longitude"] if location.get("Translation", None) is None else location["Translation"]["Longitude"]            
+                        latitude = location["Longitude"] if location.get("Translation", None) is None else location["Translation"]["Longitude"]
```

Comment:
Should this be "Latitude" rather than "Longitude"?

Reply:
done